### DISCUSSION TITLE
how can i make this legal

### DISCUSSION THREAD

**Discussion by NeoAlucardGAMER** [kwsch/PKHeX#4165] [Q&A]
![error](https://github.com/kwsch/PKHeX/assets/157064720/a73291e0-8230-4138-9d4d-b3f97a73b3f5)

Hi everyone i catch this pokemon  in Violet Version but i have this problem - Unable to match an encounter from origin game - in the last update of pkhex what can i do ?

**Comment by sora10pls** [comment]
GitHub Discussions are not for assistance in making Pokémon. There are a number of guides online you can refer to (use the encounter database, don’t make Pokémon from scratch).

Chances are your specific Kleavor was obtained from a Tera Raid, which has RNG correlation, meaning you can not edit immutable values without making it illegal.

**Reply by NeoAlucardGAMER** [reply]
i got it in violet in the Terarium

**Comment by Lusamine** [comment]
If you caught this Pokémon in an unmodified game, made NO edits, and it is flagging illegal, then create an [issue](https://github.com/kwsch/PKHeX/issues) and upload the file there.

If you made any edits to the Pokémon, it is as @sora10pls said -- seek out guides to learn how to fabricate Pokémon. Project Pokémon for example, has forums where you can ask for help.

**Reply by NeoAlucardGAMER** [reply]
i  caught on dlc 2

**Reply by Lusamine** [reply]
Then do as I told you about creating an issue. Posting/replying here won't actually get the issue fixed.
